src/lib/products.ts

```diff
@@ -82,6 +83,18 @@ export const products: { [product in Product]: ProductData } = {
 		github: "https://github.com/leaningtech/cheerpx-flash",
 		repositoryName: "cheerpx-flash",
 	},
+	"cheerpj-jnlprunner": {
+		id: "cheerpj-jnlprunner",
+		name: "CheerpJ JNLP Runner",
+		href: "/cheerpj-jnlprunner",
+		logotype: jnlprunnerLogotype,
+		favicon: "/cheerpj/favicon.png",
```

Comment:
This favicon doesn't seem to be working, possibly? In any case we should definitely be using the JNLP runner's icon instead of the standard CheerpJ one!